### DISCUSSION TITLE
Sent iota token by mistake with bep20

### DISCUSSION THREAD

**Discussion by Silvanus357** [iotaledger/firefly#7941] [Help & Support]
I sent my iota tokens from gate.io to trust wallet with bep20 network. I didn’t know that i had to sent with iota network. I contacted Discord support team but couldn’t connect my wallet due to problem with server but have them my txid. They said my 36 hours is finished and i couldn’t contact them again but i check my bsc scan there was transaction happened after i gave my transaction hash. I don’t know where it has gone. I can not connect with them with same email therefore i registered with my other email yenisinemaci@gmail.com and i managed connect my wallet. Unfortunately the support team is not very helpful at all. It has been 3 days and same guy appears, his name is Alex. He keeps writing  i didn’t connect my wallet. If you check my details you will see i connect my wallet. I start worrying because support team is not working properly in Discord. How can i contact support team anywhere else or Discord?

**Comment by begonaalvarezd** [comment]
This is the firefly repository and we are a L1 wallet, we dont support the bep20 network 🙏🏼 . I dont see how we can help you, maybe you can join the official IOTA discord and see if someone experienced a similar problem, we have a `#help` channel you can try posting into https://discord.iota.org/
